streams/src/main/java/org/apache/kafka/streams/state/KeyValueStore.java

```diff
@@ -97,4 +97,11 @@
      */
     KeyValueIterator<K, V> all();
 
+    /**
+     * Return the number of key-value mappings in this store.
+     *
+     * @return the number of key/value pairs in the store.
+     */
+    int size();
```

Comment:
It seems like this method doesn't return an exact `size` for the RocksDb implementation so we need to spec it differently if we want to include it. We should also consider the right name and whether we actually want to expose it.

Let's see what @guozhangwang thinks.